lib/msf/core/modules/loader/executable.rb

```diff
@@ -82,6 +82,11 @@ def read_module_content(parent_path, type, module_reference_name)
       load_error(full_path, Errno::ENOENT.new)
       return ''
     end
-    Msf::Modules::External::Shim.generate(full_path)
+    begin
+      Msf::Modules::External::Shim.generate(full_path)
+    rescue ::Exception => e
+      elog "Unable to load module #{full_path} #{e.class} #{e}"
+      ''
```

Comment:
We will also want to generate a `load_error` here so that framework knows something went wrong with the module.